recipes/r-admm/meta.yaml

```
@@ -0,0 +1,102 @@
+{% set version = '0.3.3' %}
+{% set posix = 'm2-' if win else '' %}
+{% set native = 'm2w64-' if win else '' %}
+
+package:
+  name: r-admm
+  version: {{ version|replace("-", "_") }}
+
+source:
+  url:
+    - {{ cran_mirror }}/src/contrib/ADMM_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/ADMM/ADMM_{{ version }}.tar.gz
+  sha256: 1618e822808c68ec76f9060dc39c6ac2cc6f5dff0603a0359f727490bf8d7111
+
+build:
+  merge_build_host: True  # [win]
+  number: 0
+  rpaths:
+    - lib/R/lib/
+    - lib/
+  missing_dso_whitelist:
+    - $RPATH/R.dll
+    - $RPATH/Rlapack.dll
+    - $RPATH/Rblas.dll
```

The preferred approach to this is done in this PR: https://github.com/conda/conda-build/pull/4786

```suggestion
    - '*/R.dll'        # [win]
    - '*/Rlapack.dll'  # [win]
    - '*/Rblas.dll'    # [win]
```